README.md

```diff
@@ -95,17 +52,20 @@ For instance,
 ```python
 import logging
 for logger_name in ('snowflake.snowpark', 'snowflake.connector'):
-   logger = logging.getLogger(logger_name)
-   logger.setLevel(logging.DEBUG)
-   ch = logging.StreamHandler()
-   ch.setLevel(logging.DEBUG)
-   ch.setFormatter(logging.Formatter('%(asctime)s - %(threadName)s %(filename)s:%(lineno)d - %(funcName)s() - %(levelname)s - %(message)s'))
-   logger.addHandler(ch)
+    logger = logging.getLogger(logger_name)
+    logger.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    ch.setFormatter(logging.Formatter('%(asctime)s - %(threadName)s %(filename)s:%(lineno)d - %(funcName)s() - %(levelname)s - %(message)s'))
+    logger.addHandler(ch)
 ```
 
 ## Samples
 The [Developer Guide][developer guide] and [API references][api references] have sample code.
```

Comment:
@iamontheinet Do we already have a link for the sample files? We can link here. It's no problem that it's private now.